### PR TITLE
[DevGuide] Crash filtering details

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Comment by turtledreams** [comment]
awesome, we might expand it a bit with some

**Comment by turtledreams** [comment]
Adding the whole team for review because we all will implement this feature in our SDKs. So we better check if everything is clear for us and we can plan or discuss details starting from here.

File: sdk_dev_guide/current.md [hunk 1]
```diff
@@ -472,6 +472,69 @@ function getUniqueMsTimestamp(){
...
+Countly.sharedInstance().init(config);</code></pre>
+<p>
+  This callback will take a "CrashData" object containing all crash-related data
+  and modify or omit it. If "true" is returned from the callback, the crash must
+  be omitted.
+</p>
+<pre><code class="java">interface CrashFilterCallback {
+    boolean filterCrash(CrashData crash);
+}</code><code class="java"></code></pre>
+<p>The crash data object must contain:</p>
```
Comment (AliRKat) [inline sdk_dev_guide/current.md:491]: I think we should turn these into bulleted list

File: sdk_dev_guide/current.md [hunk 2]
```diff
@@ -472,6 +472,69 @@ function getUniqueMsTimestamp(){
...
+<p>
+  Setters mustn't accept invalid values like null, undefined, etc.
+</p>
+<p>
+  Also, there must be a way to report which fields have been modified for the server.
+</p>
+<p>
+  A metric for crashes will be calculated as an integer of "_ob" bits. If some
+  fields changed while filtering, bits must be set as:
+</p>
```
Comment (AliRKat) [inline sdk_dev_guide/current.md:515]: Also, this section should be bulleted list in my opinion

**Review by AliRKat — CHANGES_REQUESTED** [review]
Content-wise perfect work 🚀 

Visual-wise, I feel the language is clear and easy to read. My only suggestion is to turn the normally written lists into bulleted lists. Bulleted lists are much easier to review and wouldn't confuse them with other sections.

**Review by peterBrxwn — APPROVED** [review]

**Review by AliRKat — APPROVED** [review]
perfecto 👌

**Review by turtledreams — APPROVED** [review]

**Pull request merged**